lambdas/postcode-lookup/src/main/java/uk/gov/di/ipv/cri/address/api/handler/PostcodeLookupHandler.java

```diff
@@ -41,24 +55,37 @@ public PostcodeLookupHandler() {
     public APIGatewayProxyResponseEvent handleRequest(
             APIGatewayProxyRequestEvent input, Context context) {
 
+        String sessionId = input.getHeaders().get(SESSION_ID);
+        eventProbe.addDimensions(Map.of(SESSION_ID, sessionId));
```

Comment:
These dimensions allows us to partition our metrics in CloudWatch. I don't think we need to add any dimensions to the eventProbe for this lambda. But I think we are missing pushing a counter metric for the happy case i.e. `eventProbe.counterMetric("postcode_lookup");` and also pushing that same metric for all exception cases i.e. `eventProbe.log(INFO, e).counterMetric("postcode_lookup", 0d);`

This will allow us to dashboard/alert on the success rate of this lambda

Reply:
Resolved with latest changes